spec.md

```diff
@@ -436,25 +476,30 @@ Not all permutations of plan changes are expected to be supported. For example,
 #### Parameters
 | Parameter name | Type | Description |
 | --- | --- | --- |
-| accepts_incomplete | boolean | A value of true indicates that the marketplace and its clients support asynchronous broker operations. If this parameter is not included in the request, and the broker can only provision an instance of the requested plan asynchronously, the broker SHOULD reject the request with a `422 Unprocessable Entity` as described below. |
+| accepts_incomplete | boolean | A value of true indicates that the marketplace and its clients support asynchronous broker operations. If this parameter is not included in the request, and the broker can only provision a service instance of the requested plan asynchronously, the broker SHOULD reject the request with a `422 Unprocessable Entity` as described below. |
 
 ##### Body
 
 | Request Field | Type | Description |
 | --- | --- | --- |
+| context | object | Contextual data under which the service instance is created. |
 | service_id* | string | The ID of the service (from the catalog). MUST be globally unique. MUST be a non-empty string. |
 | plan_id | string | The ID of the plan (from the catalog) for which the service instance has been requested. MUST be unique to a service. If present, MUST be a non-empty string. |
 | parameters | JSON object | Configuration options for the service instance. An opaque object, controller treats this as a blob. Brokers SHOULD ensure that the client has provided valid configuration parameters and values for the operation. |
-| previous_values | object | Information about the instance prior to the update. |
-| previous_values.service_id | string | ID of the service for the instance. If present, MUST be a non-empty string. |
+| previous_values | object | Information about the service instance prior to the update. |
+| previous_values.service_id | string | Deprecated; determined to be unnecessary as the value is immutable. ID of the service for the service instance. If present, MUST be a non-empty string. |
 | previous_values.plan_id | string | ID of the plan prior to the update. If present, MUST be a non-empty string. |
-| previous_values.organization_id | string | ID of the organization specified for the instance. If present, MUST be a non-empty string. |
-| previous_values.space_id | string | ID of the space specified for the instance. If present, MUST be a non-empty string. |
+| previous_values.organization_id | string | Deprecated as it was redundant information. ID of the organization specified for the service instance. If present, MUST be a non-empty string. |
```

Comment:
Actually, in this case I think we're removing it because its not used - we don't have a previous_values.context.